Application/EdFi.Ods.Api/Container/Modules/ContextStorageModule.cs

```diff
@@ -13,13 +13,10 @@ namespace EdFi.Ods.Api.Container.Modules
     public class ContextStorageModule : Module
     {
         protected override void Load(ContainerBuilder builder)
-        {
-            // Primary context storage for ASP.NET web applications
-            builder.RegisterType<HttpContextStorage>().As<IContextStorage>().AsSelf();
-
```

Comment:
My concern here will is this an issue under iis? could you please verify

Reply:
No there wont be any issue actually this line of code was commented since there were no usages of HttpContextStorage object except in tests, we can still have this line of code.

Reply:
Actually there was a suggested change to remove the commented line , i have reverted the change.